firmwares/zooz/ZEN74-V01.json

```diff
@@ -0,0 +1,23 @@
+{
+	"devices": [
+		{
+			"brand": "Zooz",
+			"model": "ZEN74",
+			"manufacturerId": "0x027a",
+			"productType": "0x7000",
+			"productId": "0xa004",
+			"firmwareVersion": {
+				"min": "10.0",
+				"max": "10.255"
+			}
+		}
+	],
+	"upgrades": [
+		{
+			"version": "10.10.1",
+			"changelog": "* Includes firmware versions: 2.10, 2.20, 2.30.\n* Separated ramp rate into 4 new settings: on physical (parameter 9), on Z-Wave (parameter 28), off physical (parameter 27), and off Z-Wave (parameter 29).\n*  Improved night light mode (it takes 1 second instead of 2 to turn the night on via night light mode).\n* Added a new setting to indicate remote dimming duration when using the ZEN74 in association with other dimmers (parameter 30.",
```

Comment:
The PR title indicates that you wanted to add the region here, but I don't see it. Is that correct?